lib/glueby/contract/payment.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Glueby
+  module Contract
+    # This class can send TPC between wallets.
+    #
+    # Examples:
+    #
+    # sender = Glueby::Wallet.load("wallet_id")
+    # reciever = Glueby::Wallet.load("wallet_id")
+    #                      or
+    #            Glueby::Wallet.create
+    #
+    # Balance of sender and reciever before send
+    # sender.internal_wallet.balance
+    # => 100_000(tapyrus)
+    # reciever.internal_wallet.balance
+    # => 0(tapyrus)
+    #
+    # Send
+    # Payment.transfer(sender_wallet: sender, receiver_wallet: reciever, amount: 10_000)
+    # sender.internal_wallet.balance
+    # => 90_000
+    # reciever.internal_wallet.balance
+    # => 10_000
+    #
+    class Payment
+      include Glueby::Contract::TxBuilder
+      extend Glueby::Contract::TxBuilder
+
+      class << self
+        def transfer(sender_wallet:, receiver_wallet:, amount:, fee_provider: FixedFeeProvider.new)
+          raise Glueby::Contract::Errors::InvalidAmount unless amount.positive?
+
+          tx = Tapyrus::Tx.new
+          fee = fee_provider.fee(dummy_tx(tx))
+          utxos = sender_wallet.internal_wallet.list_unspent
+
+          sum, outputs = collect_uncolored_outputs(utxos, fee + amount)
```

Comment:
I think `collect_uncolored_outputs` would be simpler to provide as a method by InternalWallet instead of list_unspent.
@Yamaguchi what do you think?

Reply:
You mean that Glueby::Internal::Wallet should provide the method `collect_uncolord_outputs(amount)`

```ruby
class Glueby::Internal::Wallet
  def collect_uncolord_outputs(amount)
    utxos = list_unspent
    ...
  end
end

```

Reply:
I see. 
@fryota142  Can you fix this method?

Reply:
Yes! I fix it.